models/cart.model.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose")
+const { Schema } = mongoose;
+
+const { Product } = require('./product.model');
```

Comment:
what the use of the line? if you are including this because you are referring product, you can remove this line.

Reply:
Oh yeah.I removed it.